src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -487,8 +499,9 @@ private static VersionInfo GetVersionInfo()
                 loaderAssemblyName: typeof(FeatureWeightsCalibratedModelParameters<TSubModel, TCalibrator>).Assembly.FullName);
         }
 
-        private FeatureWeightsCalibratedModelParameters(IHostEnvironment env, ModelLoadContext ctx)
-            : base(env, RegistrationName, GetPredictor(env, ctx), GetCalibrator(env, ctx))
+        internal FeatureWeightsCalibratedModelParameters(IHostEnvironment env, ModelLoadContext ctx,
```

Comment:
>internal [](start = 8, length = 8)

Does this need to be internal?

Reply:
No. This was a mistake. I've fixed this.